float64[25] local_x
float64[25] local_y
float64[25] local_theta
float64[25] world_x
float64[25] world_y
